src/api/app/models/workflow_token_user.rb

```diff
@@ -0,0 +1,4 @@
+class WorkflowTokenUser < ApplicationRecord
+  belongs_to :user
+  belongs_to :token, class_name: 'Token::Workflow'
+end
```

Comment:
In general, I miss the relationship with OBS groups as well.

Reply:
We'll tackle that later